code/modules/clothing/masks/mystic.dm

```diff
@@ -52,20 +52,26 @@
 	flick("happiest_flash", src)
 	canremove = 1
 
+var/list/has_been_shade
```

Comment:
Should I always initialize global lists in their definition? I thought it was better for performance for them to not be initialized until they are needed.